Sourabh_Kanwade/html/Assignment 6/Question_1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Forms</title>
+        <link rel="stylesheet" href="./question_1.css" />
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <hr />
+        <h2>Personal Details</h2>
+        <label for="first-name">First Name: </label>
+        <input type="text" name="fname" id="first-name" placeholder="First Name" />
+        <br />
+        <br />
+        <label for="last-name">Last Name: </label>
+        <input type="text" name="lname" id="last-name" placeholder="Last Name" />
+        <br />
+        <br />
+        <label>Gender : </label>
+        <input type="radio" id="male" name="gender" value="male" />
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="gender" value="female" />
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="gender" value="other" />
+        <label for="other">Other</label>
+        <br />
+        <br />
+        <label>Age : </label>
+        <input type="radio" id="less-18" name="age" />
+        <label for="less-18">Male</label>
+        <input type="radio" id="between-18-25" name="age" checked />
+        <label for="between-18-25">18-25</label>
+        <input type="radio" id="between-25-40" name="age" />
+        <label for="between-25-40">25-40</label>
+        <h2>Academics Details</h2>
+        <input type="checkbox" name="skill-1" id="skill-1" value="HTML" />
+        <label for="skill-1">HTML</label>
+        <input type="checkbox" name="skill-2" id="skill-2" value="Java Script" />
```

Comment:
It should me end on same line

Reply:
Formatted by prettier extension So I think it is Ok.